Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM golang:alpine AS builder
+RUN apk update && apk add --no-cache git
+WORKDIR $GOPATH
+RUN cd src && git clone https://github.com/tektoncd/experimental && cd experimental/octant-plugin && \
```

Comment:
hey, why not downloading tekton rather than rebuilding it? for instance: https://github.com/sunix/che-openshift-tekton/blob/master/Dockerfile

Reply:
@gattytto did you have this for a particular reason? If not we can just download the latest major release.

Reply:
@ericwill could you try
```
FROM quay.io/eclipse/che-sidecar-kubernetes-tooling:1.0.9-a40b3bb
RUN wget https://github.com/tektoncd/cli/releases/download/v0.10.0/tkn_0.10.0_Linux_x86_64.tar.gz  && \
    tar xvzf tkn_0.10.0_Linux_x86_64.tar.gz -C /usr/local/bin/ tkn && \
    rm tkn_0.10.0_Linux_x86_64.tar.gz

```

Reply:
Sure, but we have a newer version of the che-sidecar-kubernetes-tooling image available (is that okay)?

Reply:
ok use the latest of kubernetes tooling + 0.9.0 insteand of 0.10.0 for tkn

Reply:
Understood, will update the PR this afternoon